backend/ingestion/extract_census_data.py

```diff
@@ -1 +1,147 @@
- 
\ No newline at end of file
+import os
+import sys
+import csv
+import logging
+import requests
+from typing import List, Dict
+from dotenv import load_dotenv
+
+########################################################################################
+# SETUP
+########################################################################################
+
+# Enviornment variables
+load_dotenv()
+CENSUS_API_KEY = os.getenv("CENSUS_API_KEY")
+
+# Logging configuration
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+# Mappings
+variable_ids = {"B01001_001E": "total_population"}
+city_fips = {
+    "nyc": {"state_fips": "36", "county_fips": ["061", "047", "081", "005", "085"]},
+    "chicago": {"state_fips": "17", "county_fips": ["031"]},
+    "portland": {"state_fips": "41", "county_fips": ["051"]},
+}
+
+########################################################################################
+# HELPER FUNCTIONS
+########################################################################################
+
+
+def valid_command_line_arg(supported_cities: List[str]) -> str:
```

Comment:
just fyi another cool command line tool is [click](https://click.palletsprojects.com/en/8.1.x/)